docs/examples.rst

```diff
@@ -53,9 +58,22 @@ Example: synthesizing 1000 soundscapes in one go
     for n in range(n_soundscapes):
 
         print('Generating soundscape: {:d}/{:d}'.format(n+1, n_soundscapes))
+        # generate a random seed for this Scaper object
+        # because of the earlier seed these will always be the same and are given to the
+        # Scaper object for reproducible results.
+
+        seed = np.random.randint(0, 100000)
+
+        ## alternate ways to define random state:
+        #     seed = np.random.RandomState(0)
+        #
+        # or don't define any random state. runs will be random and not reproducible in 
+        # this case. you can use np.random.get_state() to reproduce the run after the fact
+        # if needed:
+        #     seed = None
 
         # create a scaper
-        sc = scaper.Scaper(duration, fg_folder, bg_folder)
+        sc = scaper.Scaper(duration, fg_folder, bg_folder, random_state=seed)
```

Comment:
This example is great, but, as discussed, we might want to update it if we add a function along the lines of `Scaper.reset_event_spec()`